cmd/experimental/kjobctl/pkg/cmd/create/create.go

```diff
@@ -129,6 +155,10 @@ func NewCreateCmd(clientGetter util.ClientGetter, streams genericiooptions.IOStr
 		"Request is a set of (resource name, quantity) pairs.")
 	cmd.Flags().StringVar(&o.LocalQueue, localQueueFlagName, "",
 		"Kueue localqueue name which is associated with the resource.")
+	cmd.Flags().DurationVar(&o.PodRunningTimeout, podRunningTimeout, podRunningTimeoutDefault,
+		"The length of time (like 5s, 2m, or 3h, higher than zero) to wait until at least one pod is running. Default: 1m")
+	cmd.Flags().BoolVar(&o.RemoveInteractivePod, "rm", false,
+		"Remove pod when interactive session exits. Default: false")
```

Comment:
Could you please add on description that it's only for interactive mode?

Reply:
Done-